scripts/build-initrd.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+source scripts/common.sh
+
+readonly BUSYBOX_VER=1.34.0
+readonly BUSYBOX_DL="https://busybox.net/downloads/busybox-${BUSYBOX_VER}.tar.bz2"
+readonly BUSYBOX_ARCHIVE=busybox.tar.bz2
+
+function download_busybox()
+{
+    echo "Downloading Busybox version ${BUSYBOX_VER}..."
+    curl -L -o ${OUT}/${BUSYBOX_ARCHIVE} ${BUSYBOX_DL} || exit 1
+    echo "Extracting..."
+    tar -xjf ${OUT}/${BUSYBOX_ARCHIVE} -C ${OUT} || exit 1
+}
+
+function build_busybox()
+{
+   echo "Building Busybox..."
+   cp -f scripts/busybox.config ${OUT}/busybox-${BUSYBOX_VER}/.config
+   pushd ${OUT}/busybox-${BUSYBOX_VER}
+   make -j `nproc` && make install
+   cd _install
+   mv linuxrc init
+   mkdir -p etc/init.d
+   cd etc/init.d
+   echo -e "#!/bin/sh\nmount -t devtmpfs devtmpfs /dev\nmkdir -p /dev/pts\nmount -vt devpts -o gid=4,mode=620 none /dev/pts" > rcS
+   chmod 755 rcS
+   cd ../../../_install
+   find . | cpio -o --format=newc > root_fs.cpio
```

Comment:
Change this command to
```shell
find . | cpio -o --format=newc > ../rootfs.cpio
```

Always use "rootfs" instead of "root_fs" for consistency.